packages/canary-react/.storybook/preview.js

```diff
@@ -12,6 +12,13 @@ export const parameters = {
       },
       hideNoControlsWarning: true
     },
+    options: {
+      storySort: {
+        method: 'configure',
+        includeNames: true,
+        order: ['*', ['*', ['Docs', 'Playground']]],
```

Comment:
Probably out of scope for this pr but string convention is not followed in this file